ts/features/idpay/initiative/details/utils/hooks.ts

```diff
@@ -0,0 +1,47 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import React from "react";
+import { useIODispatch, useIOSelector } from "../../../../../store/hooks";
+import { idpayTimelineSelector } from "../store";
+import { idpayTimelinePageGet } from "../store/actions";
+
+export const useInitiativeTimelineFetcher = (
+  initiativeId: string,
+  pageSize: number
+) => {
+  const dispatch = useIODispatch();
+  const timelineFromSelector = useIOSelector(idpayTimelineSelector);
```

Comment:
```suggestion
  const timeline = useIOSelector(idpayTimelineSelector);
```

The cost holds the timeline not the selector.

Reply:
Alternative: potTimeline/timelinePot